server.lua

```diff
@@ -89,9 +126,32 @@ RegisterServerEvent('dream-postal:server:log', function(data)
         { ['Content-Type'] = 'application/json' })
 end)
 
+AddEventHandler("playerDropped", function()
+    if currentWorkers[source] then
+        currentWorkers[source] = nil
+    end
+
+    TriggerEvent("dream-postal:server:log", { source = source, type = "success", message = t('ended_shift')})
+end)
+
 
 -- utils
 
+function IsNearAnyDeliverPoint(source)
+    local coords = GetEntityCoords(GetPlayerPed(source))
+
+    for k,v in pairs(Config.POSTAL_DROP_OFF_PACKAGE) do
```

Comment:
not a showstopper, up to u if u want this to be done in scope of this pr

```suggestion
    for _, v in pairs(Config.POSTAL_DROP_OFF_PACKAGE) do
```